indico/modules/events/clone.py

```diff
@@ -114,11 +120,20 @@ class EventProtectionCloner(EventCloner):
     is_default = True
     uses = {'event_roles', 'registration_forms'}
 
-    def has_conflicts(self, target_event):
+    def get_conflicts(self, target_event):
+        conflicts = []
+
         if target_event.access_key != '':
-            return True
+            conflicts.append(_('The target event already contains an access key'))
+
         entries = list(target_event.acl_entries)
-        return len(entries) != 1 or entries[0].user != target_event.creator
+        if len(entries) != 1:
+            conflicts.append(_('The target event has an incompatible number of ACLs'))
+
+        if entries[0].user != target_event.creator:
+            conflicts.append(_('The target event creator is not on the ACL'))
```

Comment:
The logic here is different from what we had before. The case we want to cover (where we allow using this cloner) is when there there is exactly one ACL entry for the event creator (because that's the default in a new event).

So I'd do a check like this:

```python
if len(entries) != 1 or entries[0].user != target_event.creator:
    conflicts.append(_('The target event already has a custom ACL'))
```

Reply:
Got it, thanks :)